harvey/images.py

```diff
@@ -2,66 +2,68 @@
 import subprocess
 
 import requests
+import docker
 
 from harvey.globals import Global
 
 
 class Image:
     @staticmethod
-    def build_image(config, webhook, context=''):
+    def build_image(config, webhook, context=""):
         """Build a Docker image by shelling out and running Docker commands."""
```

Comment:
Let's update the docstring since we aren't `shelling out` anymore.

Reply:
Sure.